client/webserver/site/src/js/app.ts

```diff
@@ -203,8 +203,8 @@ export default class Application {
     this.attachCommon(this.header)
     this.attach({})
     this.updateMenuItemsDisplay()
-    // initialize browser notifications
-    ntfn.fetchBrowserNtfnSettings()
+    // initialize desktop notifications
```

Comment:
This code still manages the browser ntfn's - as well as the desktop toasts.  Renaming these functions to suggest they handle desktop notifications only isn't warranted.